src/main/webapp/app/exercises/modeling/assess/modeling-assessment.service.ts

```diff
@@ -18,12 +18,15 @@ export class ModelingAssessmentService {
 
     constructor(private http: HttpClient) {}
 
-    saveAssessment(feedbacks: Feedback[], submissionId: number, submit = false): Observable<Result> {
-        let url = `${this.resourceUrl}/modeling-submissions/${submissionId}/assessment`;
+    saveAssessment(feedbacks: Feedback[], submissionId: number, submit = false): Observable<Result | null> {
```

Comment:
Can you make this consistent to the version in other ...assessment.service.ts?
Also notice that we would like to avoid the use of `null` on the client side

Reply:
```suggestion
    saveAssessment(feedbacks: Feedback[], submissionId: number, submit = false): Observable<EntityResponseType> {
```